components/feature/searchwidget/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <color name="mozac_mobile_layer_color_1">@color/photonLightGrey10</color>
```

Comment:
@gabrielluong are these the right colour resources to use nowadays?

Reply:
I think the colors look right on a first glance. 

However, @iorgamgabriel can we import ui-colors into this and avoid duplicating the colors? We should only have a single source of truth for these color variables so that we can just change it in one place and get the expected changes. The problem right now is that the naming conventions of these colors are different from how they are defined by the design system - so if the design system changes layer color 1, we would not get the expected change for this module.

Reply:
I imported ui-colors this is how i use photonLightGrey10 . I was thinking that search-widget should  have separate id for colors that could be overridden in client app (Focus and Fenix).

Reply:
Oh, I understand the error in my thinking now. I was thinking the design system was implemented in AC but it's Fenix only. So, I would avoid that confusion by not using the layer_1 or primary text/icon naming convention.

Try something like `mozac_feature_searchwidget_text_color`

Reply:
When we use this module in Fenix, I would like that we point `mozac_feature_searchwidget_text_color ` to `fx_mobile_text_color_primary` as an example so that we aren't falling back to the AC defined colors.

Reply:
Done